importers/adium.js

```diff
@@ -0,0 +1,222 @@
+var fs          = require('fs');
+var glob        = require("glob");
+var async       = require('async');
+var Promise     = require('bluebird');
+var ProgressBar = require('progress');
+var path        = require('path');
+var program     = null;
+var parseString = require('xml2js').parseString;
+
+var RemoteStorage = require("remotestoragejs");
+var ChatMessages = require("remotestorage-module-chat-messages");
+var remoteStorage = new RemoteStorage({ modules: [ ChatMessages.default ] });
+var dailyLogs = {};
+
+var collectFiles = function() {
+  var dir = program.input;
+  var mucServer = program.adiumMuc;
+
+  if (!fs.existsSync(dir)) {
+    console.log('Input directory doesn\'t contain Adium logs (should be in ~/Library/Application\ Support/Adium\ 2.0/Users/Default/Logs/Jabber.[adium-user]/)');
```

Comment:
Not sure if the mere existence of the directory confirms that there are Adium logs in it?